packages/react-instantsearch-native/package.json

```diff
@@ -39,7 +39,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "algoliasearch": "^3.27.1",
+    "algoliasearch": "^4.0.0-beta.14",
```

Comment:
This should be something like `>= 3.27.1 < 5`.

Reply:
it can stay on v3, but install v4 on CI. That's still possible :)

Reply:
Right, also it is weird to have it as a direct dependency since it is not used in the package :thinking: